### ISSUE TITLE
GCI Experience

### ISSUE THREAD

**Issue by MohitPatni0731** [jboss-outreach/blog.jboss-outreach.org#41]
Hello World!
I am Mohit Patni from India studying in high school in 10th class. I like programming very much, in my past time I develop small games by programming. In future I want to become a software engineer and work with companies like Google, Facebook, Microsoft etc. I am first time participating Google-Code-In. I came to know about Google-Code-In from my brother who worked on a project with an open source organization in summer under the Google Summer of Code programme. Google-Code-In have tasks that are chosen by open source organizations like BRL-CAD , Jboss community etc. Jboss community of open source projects. There are several project maintained by Jboss which includes testing support over IDE , application and performance monitering to microservices and various more projects maintained by Jboss community.
The challenges vary from making a logo for their brand new app to fixing an issue on their production website. Even if don’t know programming then you can participate in Google Code-in. Google Code-in is a contest to introduce pre-university students (ages 13–17) to open source software development. Since 2010, over 4500 students from 99 countries have completed work in the contest.
Because Google Code-in is often the first experience many students have with open source, the contest is designed to make it easy for students to jump right in. Open source organizations chosen by Google provide a list of tasks for students to work on during the seven week contest period. A unique part of the contest is that each task has mentors from the organization assigned should students have questions or need help along the way.
Students choose tasks they wish to work on from the following categories: coding, documentation, training, outreach, research, quality assurance and user interface. Students earn prizes for their successful completion of the tasks.You need to complete as more challenges as you can to win this challenge There are various organization to be work with in Google-Code-In. I worked with various organizations like JBoss Community , Coala and Libre health. Firstly I did two beginner tasks for Coala and Libre Health which consisted of getting involved in community by creating accounts on GitHub, Git lab, Zulip chat, Gitter chat and introducing myself on their chats. Google Code-in 2017 opens for pre-university students to register and start claiming tasks on November 28, 2017. The contest ends on January 17, 2018. Upon successful completion of tasks, eligible students can earn prizes (digital certificates, t-shirts, hoodies) and grand prize winners will receive a trip to Google’s headquarters in Mountain View, California for themselves and a parent or legal guardian for 4 nights in June 2018.It took lot of attempts to get the tasks done as I had some problem in finding a universal username for different accounts but mentors supported me by properly guiding me through all the steps.
https://photos.app.goo.gl/5YSiZKgmwGImGsjI2
Then for JBoss community I made a promotional about Google-Code-In and Open Source.
https://youtu.be/MquDAaFtgig
I think Google-Code-In is a very good initiative by Google for teenagers. I have already completed 4 tasks and have won a cool T-shirt. Now my next target is to complete more tasks to end up in top ranks. I think Google-Code-In is a very good platform for students to show their coding skills , communication , interacting with community.Hence I enjoyed to working on Google-Code-In with various organizations and hence it was joyous ride of Google-Code-In. I am new to coding and everyday I am learning more.
Thank you!!
Mohit Patni

**Comment by SachinRammoorthy** [comment]
Hey @MohitPatni0731 . Great blog, but this is supposed to go in a file under the `_posts` folder. You can look at examples [there](https://github.com/jboss-outreach/blog.jboss-outreach.org/tree/master/_posts).

**Comment by MohitPatni0731** [comment]
_post
Hello World!
I am Mohit Patni from India studying in high school in 10th class. I like programming very much, in my past time I develop small games by programming. In future I want to become a software engineer and work with companies like Google, Facebook, Microsoft etc. I am first time participating Google-Code-In. I came to know about Google-Code-In from my brother who worked on a project with an open source organization in summer under the Google Summer of Code programme. Google-Code-In have tasks that are chosen by open source organizations like BRL-CAD , Jboss community etc. Jboss community of open source projects. There are several project maintained by Jboss which includes testing support over IDE , application and performance monitering to microservices and various more projects maintained by Jboss community.
The challenges vary from making a logo for their brand new app to fixing an issue on their production website. Even if don’t know programming then you can participate in Google Code-in. Google Code-in is a contest to introduce pre-university students (ages 13–17) to open source software development. Since 2010, over 4500 students from 99 countries have completed work in the contest.
Because Google Code-in is often the first experience many students have with open source, the contest is designed to make it easy for students to jump right in. Open source organizations chosen by Google provide a list of tasks for students to work on during the seven week contest period. A unique part of the contest is that each task has mentors from the organization assigned should students have questions or need help along the way.
Students choose tasks they wish to work on from the following categories: coding, documentation, training, outreach, research, quality assurance and user interface. Students earn prizes for their successful completion of the tasks.You need to complete as more challenges as you can to win this challenge There are various organization to be work with in Google-Code-In. I worked with various organizations like JBoss Community , Coala and Libre health. Firstly I did two beginner tasks for Coala and Libre Health which consisted of getting involved in community by creating accounts on GitHub, Git lab, Zulip chat, Gitter chat and introducing myself on their chats. Google Code-in 2017 opens for pre-university students to register and start claiming tasks on November 28, 2017. The contest ends on January 17, 2018. Upon successful completion of tasks, eligible students can earn prizes (digital certificates, t-shirts, hoodies) and grand prize winners will receive a trip to Google’s headquarters in Mountain View, California for themselves and a parent or legal guardian for 4 nights in June 2018.It took lot of attempts to get the tasks done as I had some problem in finding a universal username for different accounts but mentors supported me by properly guiding me through all the steps.
https://photos.app.goo.gl/5YSiZKgmwGImGsjI2
Then for JBoss community I made a promotional about Google-Code-In and Open Source.
https://youtu.be/MquDAaFtgig
I think Google-Code-In is a very good initiative by Google for teenagers. I have already completed 4 tasks and have won a cool T-shirt. Now my next target is to complete more tasks to end up in top ranks. I think Google-Code-In is a very good platform for students to show their coding skills , communication , interacting with community.Hence I enjoyed to working on Google-Code-In with various organizations and hence it was joyous ride of Google-Code-In. I am new to coding and everyday I am learning more.
Thank you!!
Mohit Patni

**Comment by SachinRammoorthy** [comment]
No...create a new file [here](https://github.com/jboss-outreach/blog.jboss-outreach.org/tree/master/_posts) and paste your code there. Also remember to follow the format of the other files there.

**Comment by SachinRammoorthy** [comment]
@MohitPatni0731 Please close this issue now that the PR has been created.
